<launch>
  <node name="robotiq85_gripper" type="robotiq85_gripper" 
         pkg="robotiq85_gripper" output="screen">
    <rosparam command="load" 
      file="$(find robotiq85_gripper)/config/robotiq85_gripper.yaml"/>
  </node>
</launch>
